components/ui/PageSection.vue

```diff
@@ -10,13 +10,13 @@
       <div
         class="page-section__main-content"
         :class="{
-          'page-section__main-content_alone': !this.$slots.aside
+          'page-section__main-content_alone': !$slots.aside
```

Comment:
I think `PageSection` is not used anymore. Could you double check and if that's the case, delete this file?

Reply:
Good catch!

I found in total 4 components not being used currently:

- components/ui/AppIcon.vue
- components/ui/MdContent.vue
- components/ui/PageSection.vue
- components/ui/SoftwareStack.vue

I deleted those as well 🙂

Reply:
Great!! thank you!!